x-pack/test/case_api_integration/basic/tests/cases/comments/delete_comment.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+
+import { CASES_URL } from '../../../../../../plugins/case/common/constants';
+import { postCaseReq, postCommentReq } from '../../../../common/lib/mock';
+import { deleteCases, deleteCasesUserActions, deleteComments } from '../../../../common/lib/utils';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const supertest = getService('supertest');
+  const es = getService('legacyEs');
+
+  describe('delete_comment', () => {
+    afterEach(async () => {
+      await deleteCases(es);
+      await deleteComments(es);
+      await deleteCasesUserActions(es);
+    });
+
+    it('should delete a comment', async () => {
```

Comment:
I think we should add two tests that check error handling: 1) test if the comment does not exist 2) test if the comment does not belong to the specific case

Reply:
we're actually not allowing this in the UI and could remove the endpoint entirely.... it's lucky to get one test

Reply:
That's true but users can call the API directly if they want to, so we should test the API independently of the UI, to make sure that the API is doing what we expect without considering the UI.